.goreleaser.yml

```diff
@@ -50,3 +50,12 @@ release:
     ```
     docker pull ghcr.io/conduitio/conduit:{{ .Tag }}
     ```
+nfpms:
+  - license: Apache-2.0
+    maintainer: Conduit Developers <conduit-dev@meroxa.io>
+    homepage: https://github.com/ConduitIO/conduit
```

Comment:
WDYT about https://conduit.io?

Reply:
oh was gonna ask about this one too, I sometimes feel like github gives you the docs directly, and that it's easier to have it here, but yes our homepage is `conduit.io`

Reply:
I'm slightly more inclined towards conduit.io, because of the wording (homepage), but what you mentioned about GH giving docs directly is also a great point.